src/pages/AddressPage.tsx

```diff
@@ -29,7 +29,8 @@ function AddressPage({title, address, updateAddress, isLoadingApp = true}: Addre
     const {translate} = useLocalize();
 
     // Check if country is valid
-    const {street, street2} = address ?? {};
+    const {street} = address ?? {};
+    const [street1, street2] = street ? street.split('\n') : [undefined, undefined];
```

Comment:
On updating the logic here, we also need to make it work for the WorkspaceProfileAddressPage(now WorkspaceOverviewAddressPage) component. This caused an issue https://github.com/Expensify/App/issues/68887